compact/hcal/barrel_gdml.xml

```diff
@@ -15,17 +16,17 @@
           readout="HcalBarrelHits"
           vis="HcalBarrelVis"
           env_vis="HcalBarrelEnvelopeVis"
-          rmin1="HcalBarrel_rmin"
-          rmin2="ForwardServiceGap_rmax - 5.0"
+          rmin1="HcalBarrel_rmin1"
+          rmin2="HcalBarrel_rmin2"
           rmax="HcalBarrel_rmax"
-          z1="SolenoidBarrel_length"
-          z2="HcalBarrel_length"
+          z1="HcalBarrel_length1"
+          z2="HcalBarrel_length2"
           calorimeterType="HAD_BARREL">
 
   <define>
 
-    <matrix name="xposOuter" unit="mm" coldim="1" values="23.6005"/>
-    <matrix name="yposOuter" unit="mm" coldim="1" values="1802.84"/>
+    <matrix name="xposOuter" unit="mm" coldim="1" values="18.400271"/>
+    <matrix name="yposOuter" unit="mm" coldim="1" values="1833.4834"/>
 
```

Comment:
Same here, do we have a reference for these numbers?

Reply:
These are essentially meaningless.  They are a reference location based on how the tiles are oriented in the STEP file. I can add a comment to that effect.